modules/wmco-prerequisites.adoc

```diff
@@ -7,78 +7,52 @@
 
 The following information details the supported platform versions, Windows Server versions, and networking configurations for the Windows Machine Config Operator. See the vSphere documentation for any information that is relevant to only that platform.
 
-== Supported platforms based on {product-title} and WMCO versions
+[id="wmco-prerequisites-supported-5.1.0_{context}"]
+== WMCO 5.1.0 supported platforms and Windows Server versions
 
-[cols="5",options="header"]
-|===
-|Platform
-|Supported {product-title} version
-|Supported WMCO version
-|Installer-provisioned infrastructure installation support
-|User-provisioned infrastructure installation support
-
-|Amazon Web Services (AWS)
-|4.6+
-|WMCO 1.0+
-|GA
-|
+////
+The Windows Machine Config Operator {WMCO) 5.1.0 supports two methods for deploying Windows worker nodes: 
 
-|Microsoft Azure
-|4.6+
-|WMCO 1.0+
-|GA
-|
-
-|VMware vSphere
-|4.7+
-|WMCO 2.0+
-|GA
-|
-|===
+* *Machine sets*. You can use `MachineSet` objects to automatically deploy worker nodes. 
+* *BYOH Windows instances*. You can use existing Bring-Your-Own-Host (BYOH) instances, which are preconfigured by the user.
++
+[NOTE]
+====
+BYOH is not supported for bare metal/provider agnostic clusters.
+====
+////
 
-== Supported platforms for Bring-Your-Own-Host (BYOH) instances based on {product-title} and WMCO versions
+The following table lists the link:https://docs.microsoft.com/en-us/windows/release-health/windows-server-release-info[Windows Server versions] that are supported by WMCO 5.1.0, based on the applicable platform. Windows Server versions not listed are not supported and will cause errors. To prevent these errors, use only an appropriate version for your platform.
 
-[cols="5",options="header"]
+[cols="3,7",options="header"]
 |===
 |Platform
-|Supported {product-title} version
-|Supported WMCO version
-|BYOH for installer-provisioned infrastructure installation support
-|BYOH for user-provisioned infrastructure installation support
+|Supported Windows server version
 
 |Amazon Web Services (AWS)
-|4.8+
-|WMCO 3.1+
-|GA
-|
+|Windows Server 2019
 
 |Microsoft Azure
-|4.8+
-|WMCO 3.1+
-|GA
-|
+|Windows Server 2019
 
 |VMware vSphere
-|4.8+
-|WMCO 3.1+
-|GA
-|
-
-|Provider agnostic (Platform: none)
-|4.8+
-|WMCO 3.1+
-|
-|GA
+a|* Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2022 with the Windows link:https://support.microsoft.com/en-us/topic/april-25-2022-kb5012637-os-build-20348-681-preview-2233d69c-d4a5-4be9-8c24-04a450861a8d[KB5012637] patch.
```

Comment:
Can we drop the `Windows Server Long-Term Servicing Channel (LTSC)` and `Windows Server Semi-Annual Channel (SAC)` designations?

Reply:
Those designations have been in there since the start. Is there a compelling technical reason to remove?

Reply:
From the [Windows Server page](https://docs.microsoft.com/en-us/windows/release-health/windows-server-release-info):
>Windows Server is moving to the Long-Term Servicing Channel (LTSC) as our primary release channel. The Current Semi-Annual Channel (SAC) release of Windows Server, version 20H2 will be supported through its mainstream support end date of August 9, 2022.

Since there will only be LTSC going forward after 20H2, including the designation is less relevant.  Additionally, having the full text of the channel in the version table has led to confusion about supported versions because of the readability of the table.  Consistently listing the version without the channel provides a clear reference to supported versions.

Reply:
Done and done!